app/javascript/components/search/results/StudySearchResult.jsx

```diff
@@ -212,6 +215,16 @@ export function logSelectSearchResult(study, logProps={}) {
   // This is needed to easily analyze changes in click-through rate (CTR).
   logProps.numSearches = numSearches
 
+  // Number of selections done for the current search.  Users can click
+  // multiple results for a given search, e.g. via "Open in new tab", and this
```

Comment:
Should this comment be updated to reflect that this is only tracked with keyboard commands not the mouse ones for new tab? (if i've understood the rest of the conversation in this PR correctly that is)

Reply:
Good catch!  Done in 7695326.